main_exercise/server/routes/api.js

```diff
@@ -0,0 +1,36 @@
+//Define your endpoints here (this is your "controller file")
+//
+// import express from "express";
+const express = require("express");
+// import ItemManager from "../services/item_manager.js";
+const ItemManager = require("../services/item_manager.js");
+
+const router = express.Router();
+
+router.use(async (req, res, next) => {
+    console.log("A request was made to the API");
+    console.log(req.body)
+    next();
+});
+const itemManager = new ItemManager();
+router.get('/todos', async (req, res) => {
+    res.send(await itemManager.getTodoList(req,res));
+});
+router.post('/todo', async (req, res) => {
```

Comment:
```suggestion
router.post('/todos', async (req, res) => {
```
https://www.codecademy.com/article/what-is-crud